Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -109,17 +109,32 @@ class ReaderModeBarView: UIView {
         delegate?.readerModeBar(self, didSelectButton: added ? .RemoveFromReadingList : .AddToReadingList)
     }
 
+    private func updateUnread(unread: Bool) {
+        var buttonType: ReaderModeBarButtonType = unread ? .MarkAsRead : .MarkAsUnread
```

Comment:
Is this logic backwards? If it's unread, don't we want to set it to .MarkAsUnread?